pyvista/plotting/tools.py

```diff
@@ -364,6 +364,7 @@ def parse_color(color, opacity=None, default_color=None):
     elif len(color) == 3:
         pass
     elif len(color) == 4:
+        opacity = color[3]  # TODO: verify/ask whether this addition doesn't break anything
```

Comment:
It seems there are also 3 tests failing because of this change (as they expect a list of length 3 as the return value instead of a list of length 4, including the opacity).
In my opinion, it makes sense to also return back the opacity if it was already present in the input argument. This way you don't have to deal with the opacity separately after this call. E.g. you can pass a default opacity value to use if it is not present in the input and use the input's opacity otherwise.
`color = parse_color(color, opacity=1)  # Will always return a list of length 4 with opacity included`
What is your opinion about this?

Reply:
> In my opinion, it makes sense to also return back the opacity if it was already present in the input argument.

Agreed.  Also, this method lacks docs.  Adding...